internal/cli/bumpver.go

```diff
@@ -0,0 +1,148 @@
+package cli
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+
+	"github.com/futurice/jalapeno/internal/cli/option"
+	"github.com/futurice/jalapeno/pkg/recipe"
+	"github.com/futurice/jalapeno/pkg/ui/changelog"
+	"github.com/spf13/cobra"
+	"golang.org/x/mod/semver"
+)
+
+type bumpVerOpts struct {
+	RecipeName    string
+	RecipeVersion string
+	ChangelogMsg  string
+	option.Common
+	option.WorkingDirectory
+}
+
+func NewBumpVerCmd() *cobra.Command {
+	var opts bumpVerOpts
+	var cmd = &cobra.Command{
+		Use:   "bumpver",
+		Short: "Bump version number for recipe",
+		Long:  "Bump version number for recipe. By default prompts user for update increment (patch/minor/major) and changelog messsage. These can also be specified directly with the -v and -m flags.",
+		Args:  cobra.NoArgs,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return option.Parse(&opts)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := runBumpVer(cmd, opts)
+			return errorHandler(cmd, err)
+		},
+		PostRun: func(cmd *cobra.Command, args []string) {
+			cmd.Root().SetContext(cmd.Context())
+		},
+		Example: `# Prompt version increment and changelog message
+jalapeno bumpver
+		
+# Directly specify version and message
+jalapeno bumpver -v v1.0.0 -m "Hello world"`,
+	}
+
+	cmd.Flags().StringVarP(&opts.RecipeName, "recipe", "r", "", "Name of the recipe to upgrade")
+	cmd.Flags().StringVarP(&opts.RecipeVersion, "version", "v", "", "New semver number for recipe")
+	cmd.Flags().StringVarP(&opts.ChangelogMsg, "message", "m", "", "Optional changelog message")
+
+	if err := option.ApplyFlags(&opts, cmd.Flags()); err != nil {
+		return nil
+	}
+
+	return cmd
+}
+
+func runBumpVer(cmd *cobra.Command, opts bumpVerOpts) error {
+	var increment string
+	newVer := opts.RecipeVersion
+	changelogMsg := opts.ChangelogMsg
+
+	if newVer != "" && !semver.IsValid(newVer) {
+		return errors.New("invalid version format, please enter valid Semantic Version")
+	}
+
+	re, err := recipe.LoadRecipe(opts.RecipeName)
+	if err != nil {
+		return err
+	}
+
+	if opts.RecipeVersion == "" {
+		cmd.Println()
+		prompt, err := changelog.RunChangelog()
+
+		if err != nil {
+			return err
+		}
+
+		increment, changelogMsg = prompt[0], prompt[1]
+		bumpedVer, err := BumpSemVer(re.Metadata.Version, increment)
+
+		if err != nil {
+			return err
+		}
+
+		newVer = bumpedVer
+	}
+
+	err = re.Metadata.Update(re, newVer, changelogMsg)
+	if err != nil {
+		return err
+	}
+
+	err = re.Save(opts.WorkingDirectory.Dir)
```

Comment:
`opts.RecipePath` should be used here after refactoring the `opts.RecipeName` parameter.